scripts/ctype_translator.py

```diff
@@ -69,15 +70,26 @@
     c_int16: 'INTEGER',
     c_int32: 'INTEGER',
     c_int64: 'INTEGER',
-    c_uint16: 'INTEGER',
-    c_uint32: 'INTEGER',
-    c_uint64: 'INTEGER',
+    c_uint16: 'INTEGER',  # note that integers in R are ALWAYS signed, so passing
```

Comment:
Can you add a reference (URL) to a place where this is described ?

Reply:
I meant reference to a page which describes why "integers in R are ALWAYS signed"